recipes/openblas/meta.yaml

```diff
@@ -0,0 +1,43 @@
+{% set version = "0.2.18" %}
+
+package:
+  name: openblas
+  version: {{ version }}
+
+source:
+  fn: openblas-{{ version }}.tar.gz
+  url: https://github.com/xianyi/OpenBLAS/archive/v{{ version }}.tar.gz
+  md5: 805e7f660877d588ea7e3792cda2ee65
+
+build:
+  number: 0
+  track_features:
+    - vc9     # [win and py27]
+    - vc10    # [win and py34]
+    - vc14    # [win and py35]
+
+requirements:
+  build:
+    - python    # [win]
+    - perl      # [win]
+    - gcc       # [unix]
+    - binutils  # [linux]
+
+  run:
+    - libgcc    # [unix]
+
+test:
+  commands:
+    - test -f ${PREFIX}/lib/libopenblas.a                            # [unix]
+    - test -f ${PREFIX}/lib/libopenblas.so                           # [linux]
+    - test -f ${PREFIX}/lib/libopenblas.dylib                        # [osx]
+    - if not exist %PREFIX%\\Library\\lib\\libopenblas.lib exit 1    # [win]
+
+about:
+  home: http://www.openblas.net/
+  license: BSD 3-Clause
+  summary: An optimized BLAS library based on GotoBLAS2 1.13 BSD version.
+
+extra:
+  recipe-maintainers:
+    - jakirkham
```

Comment:
> Sure - thanks!

Of course. :smile:

> I didn't know about the DYNAMIC_ARCH option - that sounds like just what we need to replace MKL.

Yeah, sorry, been a bit swamped today. I was meaning to come back and respond to your more recent comment.

That is the moral of the story though. This option is key to having a performant and portable version of OpenBLAS. This has worked very nicely in my experience. The net effect is a BLAS that has fast, portable, and has very permissive licensing so it can be used with GPL'd code if needed. My understanding of MKL is the latter is not possible, but IANAL.